docs/tutorials/basic_commands.md

```diff
@@ -1,23 +1,19 @@
 # Getting Started with Ubuntu Pro Client
 
-The Ubuntu Pro Client (`pro`) provides users with a simple mechanism to
-view, enable, and disable offerings from Canonical on their system. In this tutorial
-we will cover the base `pro` commands that allow a user to successfully manage the offering
-on their machine.
+The Ubuntu Pro Client (`pro`) provides a simple mechanism for viewing, enabling and
```

Comment:
The rest of the tutorial speaks to the user directly, so I amended the wording here to do the same.